cmd/ingress-operator/start.go

```diff
@@ -38,26 +38,35 @@ const (
 	// operator's namespace that will hold the credentials that the operator
 	// will use to authenticate with the cloud API.
 	cloudCredentialsSecretName = "cloud-credentials"
-	// defaultTrustedCABundle is the fully qualified path of the trusted CA bundle
-	// that is mounted from configmap openshift-ingress-operator/trusted-ca.
-	defaultTrustedCABundle = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
+	// defaultShutdownFile is the fully qualified path of the trusted CA bundle
+	//that is mounted from configmap openshift-ingress-operator/trusted-ca.
+	defaultShutdownFile = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```

Comment:
I prefer the old name, `defaultTrustedCABundle`, because it better describes the meaning of the value whereas `defaultShutdownFile` describes one way in which it is used.

Reply:
>I prefer the old name, defaultTrustedCABundle, because it better describes the meaning of the value whereas defaultShutdownFile describes one way in which it is used.

As far as the program is concerned, the file is just some opaque bytes that trigger shutdown, and nothing about the file conveys any other internal meaning, right? Nothing else in the program interprets this file as a certificate...

Reply:
Nothing in our code does, but ultimately some library code is reading in the file at that path.  My point is that the name should reflect the primary meaning of the file, not just its meaning to our watch code.

If nothing else, I argue "the default shutdown file is the trusted CA bundle file" is clearer than "the default shutdown file is the file of opaque bytes at this path".  But I'm fine with merging as-is if you disagree.

Reply:
I think I totally misunderstood what you were saying originally — now I get it and agree the old name should come back

Reply:
Fixed